server/routes/user.py

```
@@ -0,0 +1,83 @@
+        db.session.add(user)
+        
+        db.session.commit()
+        
+        return jsonify({"message": "Account created successfully."}), 201
+
+    except ValueError as e:
+        return {"error": str(e)}, 400
+    except IntegrityError:
+        db.session.rollback()
+        return {"error": "Email or phone number already exists."}, 400
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)